athina/steps/parse_document.py

```
@@ -36,17 +36,34 @@
+                "verbose": self.verbose,
+                "result_type": self.output_format,
+                "file_url": self.file_url
+            }
+            prepared_body = self.prepare_dict(body, input_data)
+            file_path = prepared_body.get("file_url","")
+
+
+            if not file_path.startswith("https://"):
+                return self._create_step_result(
+                    status="error",
+                    data=f"Only HTTPS URLs are allowed for security",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)